ee/clickhouse/queries/column_optimizer.py

```diff
@@ -76,7 +76,7 @@ def properties_used_in_filter(self) -> Counter[PropertyIdentifier]:
             # If groups are involved, they're also used
             #
             # See ee/clickhouse/queries/trends/util.py#process_math
-            if entity.math == "unique_group":
```

Comment:
Are you confident in this? Could we be sending dirty filters where math_group_type is sent but not actually queries?

Reply:
Replied in the other comment.